src/ansys/mapdl/core/_commands/aux3/results_files.py

```diff
@@ -0,0 +1,211 @@
+
+        Notes
+        -----
+        Specifies the results file to be edited.
+        """
+        command = f"FILEAUX3,{fname},{ext}"
+        return self.run(command, **kwargs)
+
+    def delete(self, set_: str = "", nstart: str = "", nend: str = "", **kwargs):
+        r"""Specifies sets in the results file to be deleted before postprocessing.
+
+        Mechanical APDL Command: `DELETE <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_DELETE.html>`_
+
+        Parameters
+        ----------
+        set_ : str
+            Specifies that sets in the results file are to be deleted.
+
+        nstart : str
+            The first set in a results file to be deleted.
+
+        nend : str
+            The final set in a results file to be deleted. This field is used only if deleting more than one
+            sequential sets.
+
+        Notes
+        -----
+        :ref:`delete` is a specification command that flags sets in the results file for deletion. It should
+        be followed by a :ref:`compress` command, the corresponding action command that deletes the
+        specified sets.
+
+        The :ref:`delete` command is valid only in the results file editing processor (auxiliary processor
+        AUX3).
+        """
+        command = f"DELETE,{set_},{nstart},{nend}"
+        return self.run(command, **kwargs)
+
+    def modify(
+        self,
+        set_: str = "",
+        lstep: str = "",
+        iter_: str = "",
+        cumit: str = "",
+        time: str = "",
+        ktitle: int | str = "",
+        **kwargs,
+    ):
+        r"""Changes the listed values of the data in a set.
+
+        Mechanical APDL Command: `MODIFY <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_MODIFY.html>`_
+
+        Parameters
+        ----------
+        set_ : str
+            Set of data in results file to be modified.
+
+        lstep : str
+            The new load step number.
+
+        iter_ : str
+            The new load substep number.
+
+        cumit : str
+            The new cumulative iteration.
+
+        time : str
+            The new time/frequency value.
+
+        ktitle : int or str
+            Indicates if the set title should be modified.
+
+            * ``0`` - Keep the original title.
+
+            * ``1`` - Change the title to the title specified with the most current :ref:`title` command.
+
+        Notes
+        -----
+        Use this command to change the listed values in a data set in a results file. Using this command
+        does not change any actual model data; it affects only the values listed in the results file.
+
+        The :ref:`modify` command is valid only in the results file editing processor (auxiliary processor
+        AUX3), and, like the other AUX3 commands, it only affects the data steps index (DSI), time (TIM),
+        loadstep, substep and cumulative step iteration (LSP) records in the results file.
```

Comment:
```suggestion
        load step, sub step and cumulative step iteration (LSP) records in the results file.
```

Reply:
Thank you @MaxJPRey for your feedback! Unfortunately, the output is automatically generated from the MAPDL documentation with the PyConverter-XML2Py which means that such changes need to be modified directly within the [MAPDL documentation](https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v251/en/ans_cmd/Hlp_C_[UNDELETE](https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v251/en/ans_cmd/Hlp_C_UNDELETE.html?q=undelete).html?q=undelete).

Reply:
thank you @MaxJPRey ... for the moment we keep the docs as close as MAPDL docs as possible. Typos should be fixed upstream.